testsuite/kuadrant/policy/dns.py

```diff
@@ -25,15 +26,9 @@ def _check(obj):
 class LoadBalancing:
     """Dataclass for DNSPolicy load-balancing spec"""
 
-    default_geo: str
-    default_weight: int
-
-    def asdict(self):
-        """Custom asdict due to nested structure."""
-        return {
-            "geo": {"defaultGeo": self.default_geo},
-            "weighted": {"defaultWeight": self.default_weight},
-        }
+    defaultGeo: bool
+    geo: str
+    weight: Optional[int]
```

Comment:
i think weight is still a required field

Reply:
According to reference, looks like no :thinking: 
https://github.com/Kuadrant/kuadrant-operator/blob/main/doc/reference/dnspolicy.md#loadbalancingspec

Reply:
openshift dashboard says another thing. It might be wrong though